packages/ephemeral/writer.js

```diff
@@ -9,6 +8,7 @@ const pendingChanges = new WeakMap();
 
 module.exports = declareInjections({
   indexers: 'hub:indexers',
+  writers: 'hub:writers',
```

Comment:
If every Writer is going to need this parameter, perhaps we can give it to them automatically. They get instantiated here https://github.com/cardstack/cardstack/blob/474abbd05df120fd7e71f2c06737fc53d8a53e3a/packages/hub/schema/data-source.js#L38